content/_includes/home/contributers-and-governance.html

```diff
@@ -0,0 +1,103 @@
+*   Artem Barski, Cincinnati Children's Hospital Medical Center / University of Cincinnati College of Medicine; [https://orcid.org/0000-0002-1861-5316](https://orcid.org/0000-0002-1861-5316)
+*   John Chilton, Pennsylvania State University / Galaxy Project; [https://orcid.org/0000-0002-6794-0756](https://orcid.org/0000-0002-6794-0756)
+*   Kyle Cranmer, New York University; [https://orcid.org/0000-0002-5769-7094](https://orcid.org/0000-0002-5769-7094)
+*   Michael R. Crusoe, CWL Project Lead; [https://orcid.org/0000-0002-2961-9670](https://orcid.org/0000-0002-2961-9670)
+*   Brandi Davis Dusenbery, Seven Bridges Genomics, Inc.; [https://orcid.org/0000-0001-7811-8613](https://orcid.org/0000-0001-7811-8613)
+*   Niels Drost, Netherland eScience Center; [https://orcid.org/0000-0001-9795-7981](https://orcid.org/0000-0001-9795-7981)
+*   Scott Edmunds, GigaScience; [https://orcid.org/0000-0001-6444-1436](https://orcid.org/0000-0001-6444-1436)
+*   Geet Duggal, DNAnexus; [https://orcid.org/0000-0003-3485-359X](https://orcid.org/0000-0003-3485-359X)
+*   Rob Finn, EMBL-EBI; [https://orcid.org/0000-0001-8626-2148](https://orcid.org/0000-0001-8626-2148)
+*   Marc Fiume, DNAstack; [https://orcid.org/0000-0002-9769-375X](https://orcid.org/0000-0002-9769-375X)
+*   Jeff Gentry, Foundation Medicine; [https://orcid.org/0000-0001-5351-8442](https://orcid.org/0000-0001-5351-8442)
+*   Kaushik Ghose, Seven Bridges Genomics, Inc; [https://orcid.org/0000-0003-2933-1260](https://orcid.org/0000-0003-2933-1260)
+*   Carole Goble, The University of Manchester; [https://orcid.org/0000-0003-1219-2137](https://orcid.org/0000-0003-1219-2137)
+*   Oliver Hofmann, University of Melbourne / bcbio-nextgen; [https://orcid.org/0000-0002-7738-1513](https://orcid.org/0000-0002-7738-1513)
+*   Hervé Ménager, Institut Pasteur; [https://orcid.org/0000-0002-7552-1009](https://orcid.org/0000-0002-7552-1009)
+*   Folker Meyer, Argonne / University of Chicago; [https://orcid.org/0000-0003-1112-2284](https://orcid.org/0000-0003-1112-2284)
+*   Tom Morris, Curoverse; [https://orcid.org/0000-0003-0435-7851](https://orcid.org/0000-0003-0435-7851)
+*   Anton Nekrutenko, The Pennsylvania State University / Galaxy Project; [https://orcid.org/0000-0002-5987-8032](https://orcid.org/0000-0002-5987-8032)
+*   Brian O'Connor, University of California Santa Cruz; [https://orcid.org/0000-0002-7681-6415](https://orcid.org/0000-0002-7681-6415)
+*   Tibor Simko, CERN, [https://orcid.org/0000-0001-7202-5803](https://orcid.org/0000-0001-7202-5803)
+*   Nihar Sheth, DNAnexus; [https://orcid.org/0000-0003-4128-4364](https://orcid.org/0000-0003-4128-4364)
+*   Stian Soiland-Reyes, University of Manchester; [https://orcid.org/0000-0001-9842-9718](https://orcid.org/0000-0001-9842-9718)
+*   James Taylor, Johns Hopkins University / Galaxy Project; [https://orcid.org/0000-0001-5079-840X](https://orcid.org/0000-0001-5079-840X)
```

Comment:
```suggestion
```
https://github.com/common-workflow-language/cwl-website/commit/46d7f48640e8e18a496cf6b4b492cde83b39f1aa#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

Reply:
We still need to remove James Taylor, alas